test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test "should get show" do
+    get users_show_url
+    assert_response :success
+  end
```

Comment:
dont need do test, so please delete all file test on  test folder

Reply:
by all file did you mean, (application_system_test_case.rb) and (test_helper.rb) and (connection_test.rb) too?

Reply:
![Screenshot from 2021-09-14 16-18-32](https://user-images.githubusercontent.com/87749730/133223949-d6772b08-5d22-4602-a2cc-cca099f0e9d1.png)
I want to ask what should I do if rails server become like this?

Reply:
I want to ask, I rebased the 4 commits and then there are conflicting files. When I resolve the conflicts there are 2 commit and I cant rebase it become fatal error.